Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestRunIndexViewController.swift

```diff
@@ -67,6 +80,9 @@ class TestRunIndexViewController: UIViewController, UITextViewDelegate {
         noStepRunStatusLabel.text = testRunStatusInProgressStr
         noStepPassButton.setImage(notSelectedPassButtonImage, for: .normal)
         noStepFailButton.setImage(notSelectedFailButtonImage, for: .normal)
+        photoToAttach = nil
+        //TODO remove this when we have setup for camera to create the image
+//        photoToAttach = UIImage.init(named: "PASS.png")
```

Comment:
If you uncomment this line you can test the uploading of the PASS.png image to a test run (or you can change it to another one of the images in the resources folder for a different image).